### ISSUE TITLE
In setting, the back button would go back to the browser page

### ISSUE THREAD

**Issue by winstonma** [plateaukao/browser#130]
This is the Settings → About EinkBro page

![Screenshot_20220626-203622841](https://user-images.githubusercontent.com/1215090/175814571-077b19f3-e21d-4b50-9b65-19b29ae720c9.jpg)

When I press the back button on the upper left hand corner, it would bring me to the browser page. But I am expecting Einkbro bringing me back to main setting page.

**Comment by plateaukao** [comment]
modified. will be included in next version.

**Comment by winstonma** [comment]
Thanks 🙏

**Comment by winstonma** [comment]
Hi @plateaukao I downloaded the einkbro nightly build and installed on my [Onyx Nova](https://onyxboox.com/boox_nova) (there is a button at the bottom that is the same as the back button on the navigation bar).

When I go to `Settings → About EinkBro` page and press twice on the back button. I would expect the app would bring me back to the browsing page. I got the following page:

![image](https://user-images.githubusercontent.com/1215090/176325462-bc2762e5-a0ab-43aa-b7b3-a1f46d292082.png)
